### PR TITLE
Adding count substring program

### PR DESCRIPTION
**Pull request by azax25547**
This program to count the number of substrings in the given string

### CODE REVIEW

File: snippets/count_substring.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+tags: list,function,intermediate,string
+---
+
+Returns the number of a substring pattern in the given string.
+
+- Get the lengths of the string and sub string
+- Loop through the length of the substring, check if the string's parted part is equals to the substring then increase the count
+
+```python
+def count_substring(string, sub_string):
```
Comment (thevirtualbuddy) [inline snippets/count_substring.md:12]: It seems there's a hard space at each line of the function's body.

**Review by thevirtualbuddy — COMMENTED** [review]
As the [snippet guidelines](https://github.com/30-seconds/30-seconds-of-python/blob/master/CONTRIBUTING.md) say "Always use soft tabs (2 spaces), never hard tabs." 
But it seems there's a hard space at each line of the function's body.

**Comment by azax25547** [comment]
> As the [snippet guidelines](https://github.com/30-seconds/30-seconds-of-python/blob/master/CONTRIBUTING.md) say "Always use soft tabs (2 spaces), never hard tabs." 
> But it seems there's a hard space at each line of the function's body.

Okk just updating the snippet with soft tabs 🤗

**Comment by azax25547** [comment]
Hi @thevirtualbuddy, Updated the Hard tabs with 2 soft spaces

**Review by Trinityyi — COMMENTED** [review]
This is a built-in method - `string.count(substring)` .

**Pull request closed without merging**